### PR TITLE
Add Cédrick Lunven

### PR DESCRIPTION
**Pull request by clun**
Voici une contribution pour 2 sessions possibles relatives au framework FF4J.

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -13892,7 +13892,37 @@ var data = {
         "cities": [
             "Paris"
         ]
-    }
+    }, {	
+		    "since": "2016-10-31",
```
Comment (dadoonet) [inline baggers.js:13896]: Can you fix your indentation here?

File: baggers.js [hunk 2]
```diff
@@ -13892,7 +13892,37 @@ var data = {
...
+        "picture": "https://avatars3.githubusercontent.com/u/726536?v=3&s=466",
+        "websites": [
+            {   "name": "Linkedin",
+                "url": "https://fr.linkedin.com/in/clunven" },
+            {   "name": "FF4J WebSite",
+                "url": "http://ff4j.org" } ],
+        "contacts": {
+          "mail": "cedrick.lunven@gmail.com",
+          "twitter": "clunven"
+         },
```
Comment (dadoonet) [inline baggers.js:13908]: Can you fix your indentation here?

File: baggers.js [hunk 3]
```diff
@@ -13892,7 +13892,37 @@ var data = {
...
+             {  "title": "Microservices, ce que FF4J peut faire pour vous",
+                "abstract": "Au travers d'un exemple concret nous ferons une revue des différentes fonctionnalités offertes sur le feature toggle d'abord mais également la gestion des properties, le monitoring, les déploiements, la consistance",
+                "tags": [ "feature toggle","feature flipping","microservice" ],
+                "lang": [ "fr" ]
+            }
+        ],
+        "cities": [
+            "Paris"
+        ]
+	   }
```
Comment (dadoonet) [inline baggers.js:13925]: Can you fix your indentation here?

File: baggers.js [hunk 4]
```diff
@@ -13892,7 +13892,37 @@ var data = {
...
+                "url": "http://ff4j.org" } ],
+        "contacts": {
+          "mail": "cedrick.lunven@gmail.com",
+          "twitter": "clunven"
+         },
+        "location": "Paris",
+        "sessions": [
+            {   "title": "Présentation de FF4J, Proposition de Feature Toggle pour Java",
+                "abstract": "Le feature toggle, ou la capacité à activer ou désactiver une fonctionnalité à chaud, est souvent associée à une astuce de développement au service du Continuous Delivery. On peut pousser du code en production qui n'est pas complètement finalisé et introduire de la dette technique... mais il peut être tellement plus utile !!!!! Après un panorama de quelques cas d'usages très concrets, je vous propose de détailler les capacités du framework FF4J, les choix d'implémentation, les références en production, les killers features ainsi que les prochaines évolutions.",
+                "tags": [ "feature toggle","feature flipping","aop", "aspect oriented programming" ],
```
Comment (dadoonet) [inline baggers.js:13913]: `"feature toggle", "feature flipping", "aop", "aspect oriented programming"` 
would be more readable

File: baggers.js [hunk 5]
```diff
@@ -13892,7 +13892,37 @@ var data = {
...
+                "lang": [ "fr" ]
+             }, 
+             {  "title": "Microservices, ce que FF4J peut faire pour vous",
+                "abstract": "Au travers d'un exemple concret nous ferons une revue des différentes fonctionnalités offertes sur le feature toggle d'abord mais également la gestion des properties, le monitoring, les déploiements, la consistance",
+                "tags": [ "feature toggle","feature flipping","microservice" ],
```
Comment (dadoonet) [inline baggers.js:13918]: `"feature toggle", "feature flipping", "microservice"`

**Review by dadoonet — APPROVED** [review]
I left some formatting comments. If you can fix them before we merge, that'd be lovely.
If not, no worries, at some point I'll run a full format of the code.

**Comment by dadoonet** [comment]
Also can you rebase on master branch?

**Comment by clun** [comment]
will do it, no pb

**Comment by dadoonet** [comment]
Ping?

**Comment by dadoonet** [comment]
Pas de News. Closing.

**Pull request closed without merging**